contracts/hermez/future_versions/HermezV2.sol

```diff
@@ -484,64 +490,96 @@ contract HermezV2 is InstantWithdrawManagerV2 {
      * @param proofA zk-snark input
      * @param proofB zk-snark input
      * @param proofC zk-snark input
-     * @param tokenID Token identifier
-     * @param amountExit Amount exit of the leaf
-     * @param amountWithdraw Amount to withdraw
+     * @param tokenIDs Token identifier
+     * @param amountExits Amount exit of the leaf
+     * @param amountWithdraws Amount to withdraw
      * @param batchNum Batch number after exit transactions has been done
-     * @param idx Index of the exit tree account
-     * @param instantWithdraw true if is an instant withdraw
+     * @param idxs Index of the exit tree account
+     * @param instantWithdraws true if is an instant withdraw
      * Events: `WithdrawEvent`
      */
-    function withdrawCircuit(
+    function withdrawMultiToken(
         uint256[2] calldata proofA,
         uint256[2][2] calldata proofB,
         uint256[2] calldata proofC,
-        uint32 tokenID,
-        uint192 amountExit,
-        uint192 amountWithdraw,
+        uint32[] memory tokenIDs,
+        uint192[] memory amountExits,
+        uint192[] memory amountWithdraws,
         uint32 batchNum,
-        uint48 idx,
-        bool instantWithdraw
+        uint48[] memory idxs,
+        bool[] memory instantWithdraws
     ) external {
-        // in case of instant withdraw assure that is available
-        if (instantWithdraw) {
+        require(
+            tokenIDs.length <= MAX_TOKEN_WITHDRAW,
+            "Hermez::withdrawMultiToken: MAX_TOKEN_WITHDRAW_EXCEED"
+        );
+
+        require(
+            tokenIDs.length == amountExits.length &&
+                tokenIDs.length == amountWithdraws.length &&
+                tokenIDs.length == idxs.length &&
+                tokenIDs.length == instantWithdraws.length,
+            "Hermez::withdrawMultiToken: SAME_ARRAY_LENGTH_REQUIRED"
+        );
+
+        uint256 nTokens = tokenIDs.length;
```

Comment:
this variable `nTokens` could be defined at start, and then could be used several times